pages/report-hazard/style.css

```diff
@@ -75,15 +73,128 @@ img {
   border-radius: 0 0 1rem 1rem;
 }
 
+/* Header */
+#report-hazard-body main {
+  height: 100%;
+}
+
+header {
+  display: none;
+}
+
+/* Back button */
+#backButton {
+  border: none;
+  background: none;
+  color: inherit;
+  cursor: pointer;
+  text-decoration: none;
+  color: var(--neutral-neutral-500, #667185);
+  margin-top: 12px;
+  margin-bottom: 12px;
+  margin-left: 16px;
+  margin-right: 16px;
+  display: flex;
+  align-items: center;
+}
+
+.underline {
+  width: 100%;
+  border-bottom: 1px solid var(--neutral-neutral-300, #d0d5dd);
+}
+
+.back-arrow {
+  height: 20px;
+  width: 20px;
+  padding: 0;
+  margin-right: 8px;
+}
+
+/* Question 2 */
+input[type='radio'] {
+  display: none;
+}
+
+#hazard-category label,
+#hazard-option-content label {
+  padding: 5px 10px;
+  cursor: pointer;
+  display: inline-block;
+  margin: 5px;
+  border-radius: 8px;
+  border: 1px solid var(--neutral-neutral-300, #d0d5dd);
+  background: var(--neutral-neutral-50, #f7f9fc);
+}
+
+#hazard-category input[type='radio']:checked + label,
+#hazard-option-content input[type='radio']:checked + label {
+  border-radius: 8px;
+  border: 1px solid var(--neutral-neutral-100, #f0f2f5);
+  background: var(--secondary-secondary-500-base, #d8e7e2);
+}
+
+.category-container {
+  display: flex;
+  align-items: center;
+}
+
+.category-icon,
+img {
+  width: 40px;
+  height: 40px;
+}
+
+.category-icon {
+  margin-left: 20px;
+}
+
+.text-container {
+  display: flex;
+  flex-direction: column;
+  margin: 20px;
+  margin-left: 16px;
+}
+
+#hazard-category-content label {
+  display: flex;
+  align-items: center;
+}
+
+.label-container {
+  padding: 20px;
+  width: 342px;
+  height: 104px;
+}
+
+.title-hazard {
+  color: var(--primary-primary-500-base, #243832);
+  /* Small/Body-1/Bold (700) */
```

Comment:
```
  /* Small/Body-1/Bold (700) */
  /* font-family: Instrument Sans; */
  font-size: 16px;
  font-style: normal;
  font-weight: 700;
  line-height: 150%; /* 24px */
```

These are not needed we have global classes for the text, reach out and we can review this together.

Reply:
done